Lab1_Operationalizing_Pytorch_with_Mlflow/src/infra/prov/nodes.tf

```diff
@@ -0,0 +1,108 @@
+resource "google_compute_instance" "training_node" {
+  name                      = "training-node"
+  machine_type              = "c2-standard-8"
+  tags                      = ["ssh", "web", "tf-created"]
```

Comment:
I wonder why the node is tagged with `web`. Maybe `training` would be more appropriate?

Reply:
You're right. Actually, I've started tagging by access protocols (bc of firewall rules) and later by roles. Forgot to put 'training' role there.

Reply:
```suggestion
  tags                      = ["ssh", "training", "tf-created"]
```